.github/workflows/build-release.yml

```diff
@@ -19,17 +19,17 @@ jobs:
         run: |
           ~/.composer/vendor/bin/autoindex
       - name: Create & upload artifact
-        uses: actions/upload-artifact@v1
+        uses: actions/upload-artifact@v3
         with:
           name: ${{ github.event.repository.name }}
-          path: ../
+          path: ${{ github.workspace }}
```

Comment:
I'm not absolutely sure but actions/upload-artifact@v3 does not allow ./ or ../ in `path`. 
I chose `${{ github.workspace }}` because it is equivalent to ../ in this yml context, and [the build result](https://github.com/PrestaShop/blockreassurance/actions/runs/4465189973) is the same as many previous PR ones (just without warning).
![blockreassurance_build_succeed_without_warning](https://user-images.githubusercontent.com/3759923/226347041-0196beb9-5d36-4217-81ac-29cad9a02119.png)
Be free to suggest other value for `path` :)